Services/IotHub/DevicesWrapper.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+
+using Microsoft.Azure.Devices;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.IotHub
+{
+    public interface IDevicesWrapper
+    {
+        void GetDevices(RegistryManager registryManager, string documentDbCollection, int documentDbPageSize);
+    }
+
+    public class DevicesWrapper : IDevicesWrapper
+    {
+        public void GetDevices(RegistryManager registryManager, string documentDbCollection, int documentDbPageSize)
+        {
+            registryManager.CreateQuery($"select * from {documentDbCollection}", documentDbPageSize);
```

Comment:
this code doesn't fetch the devices, it doesn't connect to the hub either

Reply:
replaced this with getJobsAsync function